Bluepill-cli/Bluepill-cli/main.m

```diff
@@ -97,7 +97,7 @@ int main(int argc, char * argv[]) {
         if (![config processOptionsWithError:&err] || ![config validateConfigWithError:&err]) {
             fprintf(stderr, "%s: invalid configuration\n\t%s\n",
                     basename(argv[0]), [[err localizedDescription] UTF8String]);
-            [config usage:1];
```

Comment:
It printed usage information when an invalid configuration was detected. Unfortunately, we have so many options now that printing help just obscures the error (it scrolls out of the way).